mobile/android/gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx4608m
 android.useAndroidX=true
 android.enableJetifier=true
+android.enableR8.fullMode=false
```

Comment:
Let's add an comment here why this is disable, with may be some references for relevant Github Issues. So that in future, if someone revisits this, they will have some context.